src/Riok.Mapperly/Descriptors/Mappings/MemberMappings/MemberExistingTargetMapping.cs

```diff
@@ -7,17 +7,21 @@ namespace Riok.Mapperly.Descriptors.Mappings.MemberMappings;
 /// <summary>
 /// A <see cref="IMemberAssignmentMapping"/> which maps to an existing target instance.
 /// </summary>
-public class MemberExistingTargetMapping(IExistingTargetMapping delegateMapping, GetterMemberPath sourcePath, GetterMemberPath targetPath)
-    : IMemberAssignmentMapping
+public class MemberExistingTargetMapping(
+    IExistingTargetMapping delegateMapping,
+    GetterMemberPath sourcePath,
+    GetterMemberPath targetGetterPath,
+    NonEmptyMemberPath targetMemberPath
```

Comment:
With the current naming and without updated comments it is really hard to tell whats the difference between those two paths. Maybe explain with a comment.